lib/logstash/outputs/logstash.rb

```diff
@@ -1,14 +1,14 @@
 # encoding: utf-8
 
-require 'logstash/inputs/base'
+require 'logstash/outputs/base'
 require 'logstash/namespace'
 
 require "logstash/plugin_mixins/plugin_factory_support"
 
-class LogStash::Outputs::Logstash < LogStash::Inputs::Base
+class LogStash::Outputs::Logstash < LogStash::Outputs::Base
   include LogStash::PluginMixins::PluginFactorySupport
 
-  config_name "logstash"
+  config_name "logstash_output"
```

Comment:
🚫 this defies the convention of the `config_name` matching the plugin's name as-declared in pipeline definitions.